ams/ams-server/src/main/java/com/netease/arctic/ams/server/optimize/FullOptimizePlan.java

```diff
@@ -145,8 +150,9 @@ protected boolean checkFullOptimizeInterval(long current, String partitionToPath
 
   /**
    * check whether node task need to build
+   *
    * @param posDeleteFiles pos-delete files in node
-   * @param baseFiles base files in node
+   * @param baseFiles      base files in node
```

Comment:
It seems add some extra white spaces.

Reply:
I tried Auto Format in IDEA, it added the white spaces automatically. It aligns with the`pos-delete` on the previous line.

Reply:
okay